fastime.go

```diff
@@ -8,177 +8,150 @@ import (
 	"unsafe"
 )
 
-type Fastime interface {
```

Comment:
I see you removed this, but why? Maybe we should start over and create new patch from master since it has been refactored?